assets/canary/daemonset.yaml

```diff
@@ -8,8 +8,8 @@ spec:
   template:
     spec:
       containers:
-        - name: hello-openshift-canary
-          # Image is set at runtime
+        - name: serve-healthcheck-canary
```

Comment:
Do we need to trigger canary daemonset changes when the canary container name changes? I guess not, since its most likely not possible for the container name to update without the image reference also updating. Thoughts on that?

Reply:
If you haven't already done so, check `daemonsets.yaml` in the e2e-upgrade CI job artifacts to verify that the operator updates the container name.